Makefile

```diff
@@ -172,6 +172,16 @@ skopeo:
 fakechroot:
 	if ! which fakechroot; then if [ -f /etc/redhat-release ]; then dnf -y install fakechroot; elif [ -f /etc/lsb-release ]; then sudo apt-get -y update; sudo apt-get -y install fakechroot; fi; fi
 
+$(BIN_DIR)/get_helm.sh:
+	mkdir -p $(BIN_DIR)
+	curl -fsSL -o $(BIN_DIR)/get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+	chmod 700 $(BIN_DIR)/get_helm.sh
+
+$(BIN_DIR)/helm: $(BIN_DIR)/get_helm.sh
+	HELM_INSTALL_DIR=$(BIN_DIR) $(BIN_DIR)/get_helm.sh
+
+helm: $(BIN_DIR)/helm
```

Comment:
nit: helm-install ?

Reply:
other rules to install tools have the name of the tool only (e.g. `gcov2lcov`, `skopeo`, ...). And `install` rule actually deploys the CRDs to the cluster, so I think we have spoiled the term. If it's not that important, I'd leave `helm`